From 41974689e041dbe9625a2e1435b195ed5e94d40f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Thu, 12 May 2011 13:44:26 +0200
Subject: [PATCH] Make dirname static

The `dirname' variable name clashes with libc dirname(3). This could
cause edquota to segfault if name server switch is configured to use
`db' back-end that attempts to call dirname(3).
---
 edquota.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/edquota.c b/edquota.c
index 4d247fa..0d72882 100644
--- a/edquota.c
+++ b/edquota.c
@@ -69,7 +69,7 @@ char *progname;
 int flags, quotatype;
 int fmt = -1;
 char *protoname;
-char *dirname;
+static char *dirname;
 
 void usage(void)
 {
-- 
1.7.4.4

